### ISSUE TITLE
Pre-set password?

### ISSUE THREAD

**Issue by SergeHaddad** [yankodimitrov/SwiftPasscodeLock#78]
Can we have a pre-set password that the developers sets instead of having the user set it to make the app accessible to few people?

**Comment by ziogaschr** [comment]
Yes, you can do this in your Apps' code. You can use your own implementation of the `PasscodeRepositoryType` to set the passcode if it doesn't exist
